cluster-provision/centos9/vmcli/cmd/utils/disk_test.go

```diff
@@ -0,0 +1,78 @@
+package utils_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/spf13/afero"
+
+	"kubevirt.io/kubevirtci/cluster-provision/centos9/vmcli/cmd/utils"
+)
+
+var _ = Describe("VM disk utils", func() {
+	var fs afero.Fs
+	var diskUtil *utils.DiskUtil
+
+	BeforeEach(func() {
+		fs = afero.NewMemMapFs()
+		diskUtil = utils.NewDiskUtil(fs)
+	})
+
+	Describe("Calculating the next disk path", func() {
+		BeforeEach(func() {
+			createMockFile(fs, "/random.qcow2")
+		})
+
+		When("there are no already-existing disks", func() {
+			It("returns the first disk as available", func() {
+				nextDisk, _, err := diskUtil.CalcNextDisk("/", "")
+				Expect(err).NotTo(HaveOccurred())
+				Expect(nextDisk).To(Equal("/disk01.qcow2"))
+			})
+
+			It("returns the default disk as the current one", func() {
+				_, currentDisk, err := diskUtil.CalcNextDisk("/", "")
+				Expect(err).NotTo(HaveOccurred())
+				Expect(currentDisk).To(Equal("box.qcow2"))
+			})
```

Comment:
Why do we need 2 different test to verify this? You can simply reduce to one and verify the 2 variables

Reply:
I personally like to have more "atomic" tests, but yeah I'll merge both given that they are small

Reply:
Avoiding tests duplication makes the code more compact and in general easier to fix and read, but there are exceptions of course.